src/components/NavBar.jsx

```diff
@@ -0,0 +1,31 @@
+/////Uses NavLink for routing to Home, Alle posts (/posts), and Nieuwe post (/new-post).
+////Highlights the active page with activeClassName="active". 
+///Includes icons (House, List, PlusSquare) from Phosphor Icons.
+import { NavLink } from 'react-router-dom';
+   import { House, List, PlusSquare } from '@phosphor-icons/react';
+
+   function NavBar() {
+     return (
+       <nav className="nav-bar">
+         <ul className="nav-list">
+           <li className="nav-item">
+             <NavLink to="/" activeClassName="active">
+               <House size={20} /> Home
+             </NavLink>
+           </li>
+           <li className="nav-item">
+             <NavLink to="/posts" activeClassName="active">
```

Comment:
In React router zul je de active classname moeten meegeven als callback, zie [documentatie](http://reactrouter.com/6.30.1/components/nav-link) en EdHub: https://reactrouter.com/6.30.1/components/nav-link